egs2/TEMPLATE/spk1/spk.sh

```diff
@@ -72,6 +72,27 @@ inference_config=conf/decode.yaml   # Inference configuration
 inference_model=valid.eer.best.pth  # Inference model weight file
 score_norm=false      # Apply score normalization in inference.
 qmf_func=false        # Apply quality measurement based calibration in inference.
+cos_sim=false       # Use cosine similarity for scoring
```

Comment:
I think we don't need this as we have `score_norm` and `qmf_func` to control the inference behavior. Can you let me know what this variable does?

Reply:
It's providing an alternative to what was existing for computing scores (explicit cosine similarity vs average L2 distance)
https://github.com/espnet/espnet/blob/master/espnet2/train/spk_trainer.py#L132